datacenter/ucp/2.1/guides/admin/manage-users/create-and-manage-teams.md

```diff
@@ -27,6 +27,45 @@ Then choose the list of users that you want to add to the team.
 
 ![](../../images/create-and-manage-teams-3.png){: .with-border}
 
+## Sync team members with your organization's LDAP directory.
+
+If UCP is configured to provision using by syncing with your organization's
+LDAP directory server, you will see an additional option to enable syncing the
+new team's members when creating a new team or when modifying settings of an
+existing team. [Learn how to configure integration with an LDAP directory](../configure/external-auth/index.md).
+Enabling this option will expand the form with additional field for configuring
+the sync of team members.
+
+Two methods of matching group members from an LDAP directory exist:
```

Comment:
Yeah, it seems yoda-ish because there's a really long prepositional phrase in the middle. I'll change it to:

```
Two methods exist for matching group members from an LDAP directory:
```